pkgs/development/tools/rust/rustup/default.nix

```diff
@@ -37,7 +37,11 @@ rustPlatform.buildRustPackage rec {
   postInstall = ''
     pushd $out/bin
     mv rustup-init rustup
-    for link in cargo rustc rustdoc rust-gdb rust-lldb rls rustfmt cargo-fmt cargo-clippy; do
+    binlinks=(
+      cargo rustc rustdoc rust-gdb rust-lldb rls rustfmt cargo-fmt
```

Comment:
Is there a nice way to check if we are missing a (new) link? I guess we might run into issues with future updates where new commands are being introduced that we didn't catch.

Reply:
I have not anything related to this so far. Also we might also include symlinks only available in nightly, so anything we can interfere from rustc.src might be out-dated.